portfolio/src/main/java/com/google/sps/servlets/AddCommentServlet.java

```diff
@@ -66,7 +66,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       long id = entity.getKey().getId();
       String email = (String) entity.getProperty("email");
       String comment = (String) entity.getProperty("comment");
-      long timestamp = Long.parseLong(entity.getProperty("timestamp"));
+      long timestamp = Long.parseLong((String)entity.getProperty("timestamp"));
 
       Comment curr_comment = new Comment(id, comment, email, timestamp);
```

Comment:
is this variable name changed in the other PR?

Reply:
Yes, other PR's are consistent with the camelCase style.